src/shared/components/lollipopMutationPlot/LollipopMutationPlotWrapper.tsx

```diff
@@ -0,0 +1,508 @@
+                    Color Codes
+                </span>
+                <p>
+                    Mutation diagram circles are colored with respect to the corresponding mutation types.
+                    In case of different mutation types at a single position, color of the circle is determined with
+                    respect to the most frequent mutation type.
+                </p>
+                <br/>
+                <div>
+                    Mutation types and corresponding color codes are as follows:
+                    <ul>
+                        <li>
+                            <span style={{color:MISSENSE_COLOR, fontWeight: "bold", fontSize: "14px", fontFamily:"verdana, arial, sans-serif"}}>
+                                Missense Mutations
+                            </span>
+                        </li>
+                        <li>
+                            <span style={{color:TRUNCATING_COLOR, fontWeight: "bold", fontSize: "14px", fontFamily:"verdana, arial, sans-serif"}}>
+                                Truncating Mutations
+                            </span>
+                            : Nonsense, Nonstop, Frameshift deletion, Frameshift insertion, Splice site
+                        </li>
+                        <li>
+                            <span style={{color:INFRAME_COLOR, fontWeight: "bold", fontSize: "14px", fontFamily:"verdana, arial, sans-serif"}}>
+                                Inframe Mutations
+                            </span>
+                            : Inframe deletion, Inframe insertion
+                        </li>
+                        <li>
+                            <span style={{color:OTHER_COLOR, fontWeight: "bold", fontSize: "14px", fontFamily:"verdana, arial, sans-serif"}}>
+                                Other Mutations
+                            </span>
+                            : All other types of mutations
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        );
+    }
+
+    @computed get controls() {
+        return (
+            <div>
+                <div className={this.showControls ? "fade-in" : "fade-out"}>
+                    <Form inline>
+                        <FormGroup>
+                            <Button onClick={this.handlers.handleSVGClick}>
+                                SVG
+                            </Button>
+                        </FormGroup>
+                        {' '}
+                        <FormGroup>
+                            <Button onClick={this.handlers.handlePDFClick}>
+                                PDF
+                            </Button>
+                        </FormGroup>
+                        {'  '}
+                        <FormGroup>
+                            <InputGroup>
+                                <InputGroup.Addon>Y-Axis Max:</InputGroup.Addon>
```

Comment:
We don't need to stick to that default, but it looks better to me when there are few mutations. Maybe I am too much used to it...